app/views/todo_mailer/task_pending.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1>To do pending list</h1>
+
+<table>
+    <tr>
+        <th>Title</th>
+        <th>Description</th>
+        <th>Status</th>
+        <th>Edit</th>
+        <th>Show</th>
+        <th>Delete</th>
+    </tr>
+    <tr>
+        <td colspan="4"><b>Pending homeworks</b></td>
+    </tr>
+    <% @todo_list_pending_homework.each do |todo| %>
+        <tr>
+            <td><%= todo.title %></td>
+            <td><%= todo.description %></td>
+            <td><%= todo.done_homework == true ? "Finished" : "Pending"  %></td>
+            <td><%= link_to 'Edit', edit_todo_list_path(todo) %></td>
+            <td><%= link_to 'Show', todo_list_path(todo) %></td>
+            <td><%= link_to 'Destroy', todo_list_path(todo), 
```

Comment:
The same about JS code in here, I'd suggest to remove this option from the template, the user still have the _Show_ link on it, it can be used to send it to the real project page and then he/she will see the _Destroy_ option

Reply:
Sorry,  i didn't watch this issue, it's true this is not correct use this in the mailer template